src/main/java/randoop/contract/CompareToReflexive.java

```diff
@@ -0,0 +1,66 @@
+package randoop.contract;
+
+import randoop.Globals;
+
+/**
+ * The contract: Comparing an object with itself should result in 0
```

Comment:
Add the word "reflexive" to the Javadoc.  Do the same for any other classes as appropriate.